### PR TITLE
Add more jjb templates

### PR DESCRIPTION
**Pull request by chaitanyaenr**
This commit:
- Adds jjb templates for some more scale-ci jobs to be managed by
  scale-ci-watcher.
- Updates readme.

### CODE REVIEW

**Comment by chaitanyaenr** [comment]
This is just a initial version, working on simplifying the templates - https://github.com/openshift-scale/scale-ci-pipeline/issues/6.

**Comment by chaitanyaenr** [comment]
scale-ci-linter run to validate the templates - https://travis-ci.org/openshift-scale/scale-ci-pipeline/builds/541835114?utm_source=github_status&utm_medium=notification.

File: jjb/dynamic/http-scale-test.yml [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+        if [[ $CONTAINERIZED == "true" ]]; then
+        if [[ -d "/root/scale-cd-jobs" ]]; then
+        rm -rf /root/scale-cd-jobs
+        fi
+        git clone https://github.com/chaitanyaenr/scale-cd-jobs.git /root/scale-cd-jobs
+        cd /root/scale-cd-jobs/build-scripts
+        ./controller.sh
+        else
+        rm -rf $http_test_dir
+        git clone https://github.com/jmencak/http-ci-tests.git
```
Comment (jmencak) [inline jjb/dynamic/http-scale-test.yml]: We're obsoleting this repo in favour of https://github.com/openshift-scale/workloads, I believe.  This ihttps://github.com/openshift-scale/workloads/pull/1 is very likely to merge tomorrow.

Comment (chaitanyaenr) [reply jjb/dynamic/http-scale-test.yml]: We can create a PR with the modifications, the plan is to get the existing templates into the repo and then update the existing templates for all the jobs to use workloads tooling. Will add some instructions around testing the templates on a containerized jenkins instance soon.

File: jjb/dynamic/SVT_Conformance.yml [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+          <throttleEnabled>false</throttleEnabled>
+          <throttleOption>project</throttleOption>
+          <limitOneJobWithMatchingParams>false</limitOneJobWithMatchingParams>
+          <paramsToUseForLimit />
+          </hudson.plugins.throttleconcurrents.ThrottleJobProperty>
+    publishers: []
+    triggers: []
+    wrappers:
+    - ansicolor:
+        colormap: xterm
```
Comment (mffiedler) [inline jjb/dynamic/SVT_Conformance.yml]: For conformance in 4.1 we were running openshift-tests run openshift/conformance directly.   I'll put up a 
PR to switch the wrapper script to the new test binary.

Comment (chaitanyaenr) [reply jjb/dynamic/SVT_Conformance.yml]: Sounds good, thanks @mffiedler.

File: README.md [hunk 3]
```diff
@@ -15,13 +15,20 @@ NOTE: This is also maintained at openshift/aos-cd-jobs repo [https://github.com/
 - scale-ci-linter
 
 ### Properties files
-The parameters/configuration of each Job in the scale-ci-pipeline is supplied through the properties files. It contains key=value pairs, the sample properties for all the supported jobs are hosted in scale-ci/properties.
+The parameters/configuration of each Job in the scale-ci-pipeline is supplied through the properties files. It contains key=value pairs, the sample properties for all the supported jobs are hosted in scale-ci-pipeline/properties-files.
```
Comment (akrzos) [inline README.md]: This is a suggestion to make the sentence a bit more clear:

Potentially change this:

> The parameters/configuration of each Job in the scale-ci-pipeline is supplied through the properties files. It contains key=value pairs, the sample properties for all the supported jobs are hosted in scale-ci-pipeline/properties-files.

to this

> The parameters for each job in the scale-ci-pipeline are supplied through a job specific properties file. It contains key=value pairs, sample properties for all the supported jobs are hosted in scale-ci-pipeline/properties-files.

**Comment by chaitanyaenr** [comment]
@akrzos thanks for the review, made the changes requested. PTAL.

**Pull request merged**